### PR TITLE
hide advert variants

### PR DESCRIPTION
**Pull request by webb04**
Can be set using by updating `__TEST_SPEC__` on `articleTemplate.html`

```
"tests": "{
   hideAdsTest: 0
}"
```

| id | test |
| --- | --- |
| 0 | normal/control |
| 1 | different text |
| 2 | yellow banner |

| Light | Dark |
| --- | --- |
|||
|||
|||
|||

### CODE REVIEW

File: ArticleTemplates/assets/js/boot.js [hunk 1]
```diff
@@ -96,19 +96,28 @@ const init = opts => {
             return 'default';
         };
 
+        const getHideAdsTest = (tests) => {
+            if (!tests) return 0;
+            const parsedTests = JSON.parse(tests);
```
Comment (JamieB-gu) [inline ArticleTemplates/assets/js/boot.js:101]: Occurs to me that `JSON.parse` can throw an exception, is it worth wrapping in `try...catch`?

**Comment by SLambrakis** [comment]
Very nit-picky, but there's a slight red line inside the button on the yellow banner (underneath the text). Probably not a thing any user would notice though!

**Comment by benwuersching** [comment]
@SLambrakis Aha! Very well spotted, thank you, I missed that tiny detail. Of course, once you've seen the red line you can never unsee it. Would it be possible to get rid of it?

**Comment by webb04** [comment]
Good spot, I've removed that red line now.

**Pull request merged**